nservicebus/community/index.md

```diff
@@ -150,5 +150,6 @@ These serializer packages are part of the [NServiceBusExtensions](https://github
 * [MsgPack](https://github.com/NServiceBusExtensions/NServiceBus.MsgPack)
 * [ProtoBuf-Google](https://github.com/NServiceBusExtensions/NServiceBus.ProtoBufGoogle)
 * [ProtoBuf-Net](https://github.com/NServiceBusExtensions/NServiceBus.ProtoBufNet)
+  * Fork: [ramonsmits/NServiceBus.ProtoBufNet](https://github.com/ramonsmits/NServiceBus.ProtoBufNet)
```

Comment:
@soujay why was this included in this PR?

Reply:
@adamralph yes see https://docs.particular.net/nservicebus/community/#serializers-nservicebusextensions-serializers

Reply:
@ramonsmits I don't understand your response. I know about that section. I know this line was added to it.

Reply:
OK, now that I look at it again, I guess this was added in this PR because the other changes in the PR talk about using more efficient serializers, so this list was updated.